<launch>
  <!--<include file="$(find realsense2_camera)/launch/rs_camera.launch">-->
    <!--<arg name="align_depth" value="true"/>-->
  <!--</include>-->

  <!--<include file="$(find sc_hw)/launch/sc_hw.launch"/>-->

  <node pkg="ocean_vision" name="cmt_ros" type="cmt_ros" clear_params="true" output="screen"/>

  <node pkg="ocean_vision" name="listener_mecanum" type="listener_mecanum.py" clear_params="true" output="screen">
  <remap from="/cmd_vel" to="/mobile_base/mobile_base_controller/cmd_vel"/>
    <rosparam>
        rate: 20
        max_x: 20.0
        goal_x: 0.7
        x_threshold: 0.15
        y_threshold: 0.15
        y_scale: 0
        x_scale: 1.0
        max_angular_speed: 1.0
        min_angular_speed: 0.05
        max_linear_speed: 0.8
        min_linear_speed: 0.1
        shift_pid_value: [2, 1, 0.2, 0.15]
        rotate_pid_value: [2, 1, 0.2, 0.15]
        motion_mode: 0
    </rosparam>
    <param name="config_file_path" value="($find ocean_vision)/config/pyconfig.yaml"/>
  </node>
</launch>
